src/App.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-
-import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Bookings from "./Bookings";
+import Restaurant from "./Restaurant";
+import Footer from "./Footer";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Restaurant />
+      {/* //orders={orders} handleClick={incrementCount} */}
+      <Footer
+        details={[
```

Comment:
Generally, we want our return to be as short to make it simpler to read. If the value for details was initialised above the return in a variable than doing details={exampleDetails} then your return would be shorter. Overall, same functionality before and after this change.